integrations/disco/routes/post-credential-issue/index.ts

```diff
@@ -0,0 +1,18 @@
+import { discoClient } from '@/integrations/disco/disco-client'
+
+import { IssuedCredentials } from '../../utils/types'
+
+export async function postCredentialIssue(val: { recipientDid: any; subjectData: any }) {
+  const payload = {
+    schemaUrl: 'https://raw.githubusercontent.com/discoxyz/disco-schemas/main/json/ProofOfHackCredential/1-0-0.json',
+    subjectData: { ...val?.subjectData },
+    recipientDID: val?.subjectData?.recipientDid,
+  }
+
+  const { data }: { data: IssuedCredentials } = await discoClient.post(`/credential`, JSON.stringify(payload), {
```

Comment:
you can use `useQuery` from `tanstack-query` to handle the async state management

Reply:
same as above